src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -666,7 +691,7 @@ public function blacklist($superGlobalName, $key) {
      * @param $superGlobalName string the name of the superglobal array, e.g. '_GET'
      * @return array $values without sensitive data
      */
-    private function masked(array $superGlobal, $superGlobalName) {
+    private function getMasked(array $superGlobal, $superGlobalName) {
```

Comment:
I toyed with the idea of making it protected but I think this is the kind of method that really is meant to be private